ban/tests/http/test_municipality.py

```diff
@@ -435,3 +435,13 @@ def test_cannot_change_deleted_at_with_put(put):
     assert resp.status_code == 200
     assert not models.Municipality.get(
         models.Municipality.pk == municipality.pk).deleted_at
+
+
+@authorize
+def test_authorized_responses_contain_sessions_data(get):
+    municipality = MunicipalityFactory(name="Cabour")
+    resp = get('/municipality/{}'.format(municipality.id))
+    assert resp.status_code == 200
+    session = context.get('session')
+    assert resp.headers['Session-Client'] == session.client.id
```

Comment:
This is not specific to this test: the session is saved when authenticating the request:

https://github.com/BaseAdresseNationale/ban/blob/master/ban/http/auth.py#L44